### ISSUE TITLE
Auto include button is too small

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#7087]
## Test plan:
https://github.com/brave/browser-laptop/pull/7122#issue-206072637
<hr>

**Describe the issue you encountered:** Auto include button is too small. This is the cropped picture which I took with my smartphone.

![2017-02-06 16 24 48](https://cloud.githubusercontent.com/assets/3362943/22638211/0736630a-ec8a-11e6-8d35-4479e688a33d.jpg)

The verified mark is less than 1 millimeter I think :-o

- Platform (Win7, 8, 10? macOS? Linux distro?): Windows 10

- Brave Version (revision SHA): 0.13.2 RC6

- Steps to reproduce:
    1. Open about:preferences#payments
    2. Disable "auto-include"
    2. Go to https://brianbondy.com
    
- Actual result: The icon is so small and not clear what it means

- Expected result: The icon, including the verification mark, should be enough large to make it clear what it means.

- Any related issues: #6754

**Comment by srirambv** [comment]
Can make the entire container green for verified publishers
![image](https://cloud.githubusercontent.com/assets/17010094/22645393/aabed92a-ec5e-11e6-939f-5f645be2e56c.png)

cc: @bradleyrichter

**Comment by luixxiul** [comment]
Otherwise how about making the verified icon so large as the badge: https://github.com/brave/browser-laptop/pull/7050 ?

**Comment by bradleyrichter** [comment]
The amount of green needs to be small enough to not confuse the user into thinking they are visiting a safe site and accidentally enter their sensitive information. 

The verified information is secondary and not nearly as important as whether the site is marked for funding. 

I'll try enlarging the tiny verified badge so it is just "very small" instead of "tiny".

**Comment by bradleyrichter** [comment]
new SVGs for this issue:

[fund_icons.zip](https://github.com/brave/browser-laptop/files/755412/fund_icons.zip)
